vendor/bat-native-ads/src/bat/ads/internal/segments/segments_util.cc

```diff
@@ -110,7 +116,10 @@ bool ParentSegmentsMatch(const std::string& lhs, const std::string& rhs) {
 }
 
 bool ShouldFilterSegment(const std::string& segment) {
-  DCHECK(!segment.empty());
+  if (segment.empty()) {
+    BLOG(0, "Ads segment shouldn't be empty.");
+    return true;
+  }
```

Comment:
`segment` should never be empty. Can we please instead add DCHECKs and trace the callers of this method to understand why `segment` could be empty